examples/ct/structures.go

```diff
@@ -0,0 +1,79 @@
+package ct
+
+// Code to handle encoding / decoding various data structures used in RFC 6962.
+
+import (
+	"crypto/sha256"
+	"errors"
+	"time"
+
+	"github.com/google/trillian/crypto"
+	"github.com/google/certificate-transparency/go/x509"
```

Comment:
Not sure we want to duplicate all the ct code but I don't think we've decided yet.